api/routes.py

```diff
@@ -35,6 +34,16 @@ def api_ping():
         """
         return jsonify(message="Yello World !")
 
+
+    @api.route("/api/map/historical/years/")
```

Comment:
Don't put a `/`  at the end if there is no parameter.

Reply:
Little route testing. Forget to remove that after !
Done now